src/react/projects/spark-react/src/SprkPagination/SprkPagination.js

```diff
@@ -153,47 +143,47 @@ const SprkPagination = (props) => {
           </li>
         )}
 
-        { variant === 'default' && !longVariant && totalItems / itemsPerPage > 1 && (
-          <li key={uniqueId('sprk_page_')}>
-            <SprkLink
-              onClick={e => goToPage(e, 2)}
-              href="#"
-              additionalClasses={
-                classnames(
-                  'sprk-c-Pagination__link',
-                  { 'sprk-c-Pagination__link--current': currentPage === 2 },
-                )}
-              aria-label="Page 2"
-              aria-current={currentPage === 2}
-              data-analytics={analyticsStringPage}
-            >
-              2
-            </SprkLink>
-          </li>
-        )}
-
-        { variant === 'default' && !longVariant && totalItems / itemsPerPage > 2 && (
-          <li key={uniqueId('sprk_page_')}>
-            <SprkLink
-              onClick={e => goToPage(e, 3)}
-              href="#"
-              additionalClasses={
-                classnames(
-                  'sprk-c-Pagination__link',
-                  { 'sprk-c-Pagination__link--current': currentPage === 3 },
-                )}
-              aria-label="Page 3"
-              aria-current={currentPage === 3}
-              data-analytics={analyticsStringPage}
-            >
-              3
-            </SprkLink>
-          </li>
-        )}
+        {variant === 'default' &&
+          !longVariant &&
+          totalItems / itemsPerPage > 1 && (
+            <li key={uniqueId('sprk_page_')}>
+              <SprkLink
+                onClick={e => goToPage(e, 2)}
+                href="#nogo"
```

Comment:
is there a difference between `href="#"` and `href="#nogo"`?

Reply:
href="#" will add the preventDefault() function and keep the pagination functionality from firing. href="#nogo" keeps the functionality intact.